contracts/interactors/AaveInteractor.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity =0.8.26;
+
+import '@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol';
+import {WadRayMath} from '@aave/core-v3/contracts/protocol/libraries/math/WadRayMath.sol';
+import '@uniswap/v3-periphery/contracts/libraries/TransferHelper.sol';
```

Comment:
I was using openzeppelin `SafeErc20` lib for approves, transfers etc.
Let's discuss which one of them to use (for code consistency)